### PR TITLE
Add more stats in the block cache trace analyzer

### PR DESCRIPTION
**Pull request by HaoyuHuang**
This PR adds more stats in the block cache trace analyzer.

### CODE REVIEW

File: tools/block_cache_trace_analyzer.cc [hunk 1]
```diff
@@ -456,40 +513,49 @@ void BlockCacheTraceAnalyzer::PrintStatsSummary() const {
     print_break_lines(/*num_break_lines=*/3);
     fprintf(stdout, "Statistics for column family %s:\n", cf_name.c_str());
     fprintf(stdout,
-            "Number of files:%" PRIu64 "Number of blocks: %" PRIu64
-            "Number of accesses: %" PRIu64 "\n",
+            " Number of files:%" PRIu64 " Number of blocks: %" PRIu64
```
Comment (riversand963) [inline tools/block_cache_trace_analyzer.cc]: White space intended?

File: tools/block_cache_trace_analyzer.cc [hunk 2]
```diff
@@ -27,6 +27,9 @@ DEFINE_string(
     "cache_name,num_shard_bits,cache_capacity_1,...,cache_capacity_N. "
     "cache_name is lru. cache_capacity can be xK, xM or xG "
     "where x is a positive number.");
+DEFINE_int32(block_cache_trace_downsample_ratio, 1,
```
Comment (riversand963) [inline tools/block_cache_trace_analyzer.cc]: Maybe explicitly call out the meaning of this value, e.g. sample one in every `block_cache_trace_downsample_ratio` accesses?

File: tools/block_cache_trace_analyzer.cc (hunk 1 above)
Comment (HaoyuHuang) [reply tools/block_cache_trace_analyzer.cc]: Yes. It is intended.

File: tools/block_cache_trace_analyzer.cc (hunk 2 above)
Comment (HaoyuHuang) [reply tools/block_cache_trace_analyzer.cc]: Sure. I updated the comments.

**Pull request closed without merging**